.github/workflows/run-validation.yml

```diff
@@ -0,0 +1,67 @@
+name: Run validation with RoboValidate
+
+on:
```

Comment:
There may be more, but here is one I found 'Plain Language Web Writing Tips'. The body has HTML but the text was not pasted into field when 'Source' was selected, therefore all the HTML will not be used, it will just print out as is.